responses/__init__.py

```diff
@@ -503,6 +503,11 @@ def __init__(self, headers):
     def isclosed(self):
         return True
 
+    def close(self):
+        raise ChunkedEncodingError(
+            "Connection broken: Incomplete Read, Content-length mismatch with body"
+        )
```

Comment:
Won't raising an error here cause all requests to fail? The issue proposed having close() be a no-op, but without test coverage on this it is hard to tell what is working/not working.